src/Workspaces/Core/Portable/CodeFixes/CodeFixProvider.cs

```diff
@@ -45,6 +45,7 @@ internal CodeActionRequestPriority RequestPriority
             get
             {
                 var priority = ComputeRequestPriority();
+                // Note: CodeActionRequestPriority.Low is reserved for IConfigurationFixProvider.
```

Comment:
do we have a 'lowest'?  i would be ok with 'lowest' being reserved for that.  i think it's reasonable for fixes/refactorings to be 'low'.

Reply:
Thanks, done now.